R/c14_date_list_basic.R

```diff
@@ -119,27 +119,53 @@ plot.c14_date_list <- function(x, ...) {
   old.par <- graphics::par(no.readonly = TRUE)
 
   # set plot layout
-  graphics::layout(matrix(c(1,2,3,3), 2, 2, byrow = TRUE))
+  graphics::layout(
+    matrix(c(1,2,4,3,3,4,5,5,5), 3, 3, byrow = TRUE),
+    widths = c(1,0.5,1.5), heights = c(0.7,0.8,1)
+  )
 
   # plot 1: text
-  header <- as.character(format(x)) %>%
-    strsplit("\n") %>% unlist %>%
-    sub("\t", "", .) %>%
-    sub("\t\t", "\t", .) %>%
-    trimws
-  graphics::plot(
-    0, type = 'n', axes = FALSE, ann = FALSE,
-    xlim = c(0, 1), ylim = c(length(header) + 1, 0)
+  graphics::par(mar = c(0, 0, 0, 0))
+  graphics::plot(0, type = 'n', axes = FALSE, ann = FALSE)
+  graphics::legend(
+    "topleft",
+    format(x) %>% gsub("\t", "", .),
+    bty = "n",
+    cex = 1.7
   )
-  for (i in 1:length(header)) {
-    if (i == 1) {
-      graphics::text(0, i, bquote(bold(.(header[i]))), adj = 0)
-    } else {
-      graphics::text(0, i, header[i], adj = 0)
-    }
+
+  # plot 2: small globe
```

Comment:
I don't get the 2nd globe. None of the datasets thus far have a global coverage. Thus, until there is a clear need for extending the plotted area it rather distracts instead of adds any additional information. If you want to be sure that `globe::globeearth()` will not be able to display all dots in the future, I think we should rather think about switching over to `globe::flatearth()` or something alike.

Reply:
Not so sure about that... And I really like the globe. So let's run with it for a while.